Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -125,61 +150,137 @@ private fun PlayerScreen(
-                        onPauseButtonClick = playerScreenViewModel::onPause,
-                        playPauseButtonEnabled = true,
-                        playing = state.playerState.episodePlayerState.isPlaying,
-                        onSeekBackButtonClick = playerScreenViewModel::onRewindBy,
-                        seekBackButtonEnabled = true,
-                        onSeekForwardButtonClick = playerScreenViewModel::onAdvanceBy,
-                        seekForwardButtonEnabled = true,
-                        seekBackButtonIncrement = SeekButtonIncrement.Ten,
-                        seekForwardButtonIncrement = SeekButtonIncrement.Ten,
-                        trackPositionUiModel = state.playerState.trackPositionUiModel,
-                    )
-                },
-                buttons = {
-                    SettingsButtons(
-                        volumeUiState = volumeUiState,
-                        onVolumeClick = onVolumeClick,
-                        playerUiState = state.playerState,
-                        onPlaybackSpeedChange = playerScreenViewModel::onPlaybackSpeedChange,
-                        enabled = true,
-                    )
-                },
-                modifier = modifier
-                    .rotaryScrollable(
-                        volumeRotaryBehavior(
-                            volumeUiStateProvider = { volumeUiState },
-                            onRotaryVolumeInput = { onUpdateVolume },
+                    },
+                    modifier = Modifier
+                        .requestFocusOnHierarchyActive()
+                        .rotaryScrollable(
+                            volumeRotaryBehavior(
+                                volumeUiStateProvider = { volumeUiState },
+                                onRotaryVolumeInput = { onUpdateVolume },
+                            ),
+                            focusRequester = focusRequester,
                         ),
-                        focusRequester = rememberActiveFocusRequester(),
-                    ),
-                background = {
-                    ArtworkColorBackground(
-                        paintable = episode?.let { CoilPaintable(episode.podcastImageUrl) },
-                        defaultColor = MaterialTheme.colors.primary,
-                        modifier = Modifier.fillMaxSize(),
-                    )
-                },
-            )
+                    background = {
+                        ArtworkImageBackground(
+                            artwork = episode?.let { CoilPaintable(episode.podcastImageUrl) },
+                            colorScheme = MaterialTheme.colorScheme,
+                            modifier = Modifier.fillMaxSize(),
+                        )
+                    },
+                )
+            }
         }
     }
 }
+
+@androidx.annotation.OptIn(UnstableApi::class)
+@Composable
+internal fun rememberPlayer(context: Context) = remember {
```

Comment:
considering moving this to a different file since this is not UI code

Reply:
Inherited this from a gerrit PR which didn't make to github (which we discussed), can we leave this an improvement for future PRs (or we can ask the owner of this code to help with this as a follow up)